### PR TITLE
Left menu extended for cli options

### PR DESCRIPTION
**Pull request by elliott-king**
*Issue #, if available:*
#631 

*Description of changes:*
I modified the comparison that checks whether or not there should be a side menu.
`If page.url >= item.url` => `if item.url >= page.url`

When on a 'cli' page, the item url contains the sdk parameter. The page url is only 'cli/[page].'

An alternative solution would be to make the page url include the sdk parameter. However, I believe that the comparison was still incorrect regardless.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: theme/_layouts/default.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
 											{% endif %}
 											<div class="feature-list">
 												{% for item in category.subs %}
-													{% if page.url contains item.url %}
```
Comment (attilah) [inline theme/_layouts/default.html]: To keep the logical order or the contains check I'd suggest to update the changes to at all occurrences:
```
{% assign cleanItemUrl = item.url | split: '?' | first %}
{% if page.url contains cleanItemUrl %}
```

**Comment by elliott-king** [comment]
I was initially hesitant to determine what should or should not be a header (and therefore a link). I wasn't sure that was my place to say. 

It seems that you are not allowed to have h5 headers in the navbar (?). It will not indent more than the previous header , and it seems to affect the following headers..?
See: 
Android -> authentication -> Using hosted ui ... -> using amazon cognito -> setting up oauth with ... 
CLI -> Usage -> Headless -> --yes flag (should be submenu of headless and everything above it)
CLI -> Graphql -> API category project structure -> common patterns -> filtering by type (should be submenu of previous header)

Additionally, an h4 header following an h2 header will be shown as if it were h3. They look fine on the page itself, but are not distinguished in the navbar.

Some thoughts: 
Several headers for GraphQL should be shortened, or downgraded.
- "Common patterns for the API category" is too long (it is cut off on my screen)
  - Every subheader for this is too long
- The same is a true for "AWS CloudFormation Template Parameters"
- Perhaps shorten them to "Common Patterns for APIs" and "Cloudformation Template"?

File: theme/_layouts/default.html (hunk 1 above)
Comment (elliott-king) [reply theme/_layouts/default.html]: Done.

File: theme/_layouts/default.html [hunk 2]
```diff
@@ -1,325 +0,0 @@
-<!DOCTYPE html>
```
Comment (kaustavghosh06) [inline theme/_layouts/default.html]: Why do we want to get rid of this file altogether?

Comment (elliott-king) [reply theme/_layouts/default.html]: Answered below.

**Comment by elliott-king** [comment]
I messed up the commit history (presumably by not force pushing), and got a bit confused. I rewound it & re-committed everything. default.html has no longer been deleted.

**Comment by kaustavghosh06** [comment]
@elliott-king We merged your changes as  a part of our latest CLI docs res-structure. Thanks for submitting a fix for the left menus.

**Comment by elliott-king** [comment]
Ah. That's too bad, but I understand.

Quick note @attilah in `default.html`, the order of the comparison was switched, instead of adding a new line of logic.

**Pull request closed without merging**